web-ui/src/main/resources/catalog/style/gn_editor.less

```diff
@@ -645,6 +645,11 @@ form.gn-tab-xml {
       }
     }
   }
+  // no padding in column
+  .col-sm-9 {
```

Comment:
It's not the best way to go.
I prefer you create a new class `nopadding` and apply it to the element you want.

Reply:
Yes, that would be a better solution. But this column isn't always used in a table, it's a pretty general column that is used throughout the editor.

I could add a class with a name like `nopadding-in-table`.